Dockerfile

```diff
@@ -0,0 +1,12 @@
+#Build stage
+FROM gradle:jdk17-corretto AS build
+WORKDIR /usr/app/
+COPY . .
```

Comment:
Use --chown=gradle:gradle when copying source code
Ensures files are owned by the correct user (gradle), preventing permission issues.
Security Improvement: Avoids running Gradle with root privileges